### PR TITLE
Cut(eos_cli_config_gen): Removing 'null' as valid value of esp integrity and encryption from ip-security module

### PR DESCRIPTION
**Pull request by MaheshGSLAB**
## Change Summary

 Removing 'null' as valid values of esp integrity and encryption from ip-security module.

## Related Issue(s)

Fixes #<ISSUE ID>

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
Removed the null as valid values of both esp integrity and encryption and added more valid values.

## How to test
NA

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by Shivani-gslab — COMMENTED** [review]
we can add test for the changes

File: python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ip_security.schema.yml [hunk 1]
```diff
@@ -89,7 +89,9 @@ keys:
                     - disabled
                     - sha1
                     - sha256
-                    - "null"  # TODO: AVD 5.0.0
+                    - sha384
```
Comment (Shivani-gslab) [inline python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ip_security.schema.yml:92]: I can not see these valid-values in EOS **[4.32.0F-36401836.4320F]** cli-

```
s1-leaf2(config-ipsec-sa)#esp integrity ?
  null    null security profile
  sha1    sha1 security profile
  sha256  sha256 security profile
  <cr>    

s1-leaf2(config-ipsec-sa)#esp integrity sha1 ?
  <cr>  

s1-leaf2(config-ipsec-sa)#esp integrity sha1 
s1-leaf2(config-ipsec-sa)#esp encryption ?
  aes128        AES - 128-bit Advanced Encryption Standard
  aes128gcm128  128 bit AES-GCM with 128 bit ICV
  aes128gcm64   128 bit AES-GCM with 64 bit ICV
  aes256        AES - 256-bit Advanced Encryption Standard
  aes256gcm128  256 bit AES-GCM with 128 bit ICV
  null          Null Encryption
  <cr> 
```

Comment (MaheshGSLAB) [reply python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ip_security.schema.yml:92]: it is there for 4.32.1F
```
wan1-dc1(config-ipsec-sa)#esp integrity ?
  md5     md5 security profile
  null    null security profile
  sha1    sha1 security profile
  sha256  sha256 security profile
  sha384  sha384 security profile
  sha512  sha512 security profile
  <cr>    

wan1-dc1(config-ipsec-sa)#esp encryption ?
  3des          Three key triple DES
  aes128        AES - 128-bit Advanced Encryption Standard
  aes128gcm128  128 bit AES-GCM with 128 bit ICV
  aes128gcm64   128 bit AES-GCM with 64 bit ICV
  aes256        AES - 256-bit Advanced Encryption Standard
  aes256gcm128  256 bit AES-GCM with 128 bit ICV
  null          Null Encryption
  <cr>
```

**Review by ClausHolbechArista — COMMENTED** [review]
Please update title to say what you are actually removing.
Also update release notes and porting guide. The old `"null"` option has been replaced with `"disabled"`.

**Review by Vibhu-gslab — APPROVED** [review]
LGTM!

File: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md [hunk 2]
```diff
@@ -240,6 +240,36 @@ As of AVD 5.0.0 the automatic conversion from both of the previous models has be
               route_map: static-to-bgp
     ```
 
+### The old "null" option for esp integrity and encryption has been replaced with "disabled" from the ip_security module
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]:
```suggestion
### `ip_security.sa_policies[].esp.integrity` and `.encryption` "null" option has been replaced with "disabled"
```

File: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md [hunk 3]
```diff
@@ -240,6 +240,36 @@ As of AVD 5.0.0 the automatic conversion from both of the previous models has be
...
+
+    ```yaml
+    # New data model
+    ip_security:
+      sa_policies:
+        name: Disabled
+        esp:
+          integrity: disabled
+          encryption: disabled
+    ```
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]: (add an extra ``` at the end. I could not add it here)
```suggestion
```diff
 ip_security:
   sa_policies:
     name: Disabled
     esp:
-      integrity: "null"
-      encryption: "null"
+      integrity: disabled
+      encryption: disabled
```

File: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md (hunk 2 above)
Comment (MaheshGSLAB) [reply ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]: updated

File: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md (hunk 3 above)
Comment (MaheshGSLAB) [reply ansible_collections/arista/avd/docs/porting-guides/5.x.x.md]: done

**Review by Shivani-gslab — APPROVED** [review]
LGTM!

**Review by ClausHolbechArista — APPROVED** [review]

**Pull request merged**